resources/lang/en/message.php

```diff
@@ -0,0 +1,698 @@
+<?php
```

Comment:
remove this file if lang is already present outside

Reply:
It has been removed in the release build, I will rebase it once the release is done.